crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/validation/validator/MeansAssessmentValidationProcessorTest.java

```diff
@@ -164,4 +168,15 @@ public void givenOutstandingAssessment_whenValidateIsInvoked_thenCorrectExceptio
                 () -> meansAssessmentValidationProcessor.validate(createMeansAssessmentRequest, AssessmentRequestType.CREATE));
         assertThat(validationException.getMessage()).isEqualTo(MSG_INCOMPLETE_ASSESSMENT_FOUND);
     }
+
+    @Test
+    public void givenInvalidFinancialAssessmentTimeStamp_whenValidateIsInvoked_thenCorrectExceptionIsThrown() {
+        when(meansAssessmentValidationService.isAssessmentModifiedByAnotherUser(
+                any(MeansAssessmentRequestDTO.class))
+        ).thenReturn(Boolean.TRUE);
+
+        ValidationException validationException = Assert.assertThrows(ValidationException.class,
```

Comment:
You're using a mix of fluent and regular assert methods here. This seems to be a problem throughout the class, but this might be a good opportunity address it.

You can use the assertThatThrownBy method instead (line 107) to ensure consistency.

Reply:
Changed to assertThatThrownBy